### PR TITLE
Create breadcrumbs atom

### PR DESCRIPTION
**Pull request by mboperator**
## Changes
- Added Breadcrumb styles
- Added Breadcrumb component
- Updated tests
- Documented Breadcrumbs in atom manifest

## Notes
:after pseudoselector wasn't happy

### CODE REVIEW

File: src/Breadcrumbs/index.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import style from './styles.scss';
+import Icon from '../Icon';
+import cx from 'classnames';
+
+export const Breadcrumbs = ({ className, styles, children }) => (
+  <nav className={`${style.breadcrumbs} ${className}`} styles={styles}>
+    {children.reduce((allChildren, child, index) => {
```
Comment (mboperator) [inline src/Breadcrumbs/index.jsx]: How do we feel about this? @remyyounes

File: src/css-framework/components/_breadcrumbs.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
+@import '../generic/type';
+@import '../settings/variables';
+
+.breadcrumbs {
+  font-size: 13px;
```
Comment (mboperator) [inline src/css-framework/components/_breadcrumbs.scss]: Should this be a variable?

File: src/Breadcrumbs/index.jsx (hunk 1 above)
Comment (remyyounes) [reply src/Breadcrumbs/index.jsx]: We've already talked about it in person but for reference:
We want to push the `className` usage as far as we can in order to allow non-JS clients to consume the same CSS.

File: src/css-framework/components/_breadcrumbs.scss (hunk 2 above)
Comment (remyyounes) [reply src/css-framework/components/_breadcrumbs.scss]: yes

**Pull request merged**